### ISSUE TITLE
EASY TASK FOR MAINTAINERS - XRay Propagator is not published to rubygems.org 

### ISSUE THREAD

**Issue by schanjr** [open-telemetry/opentelemetry-ruby#719]
Hello Fellow Rubyist, 

Upon experimenting to use JRUBY with [opentelemetry-propagator-xray](https://github.com/open-telemetry/opentelemetry-ruby/tree/main/propagator/xray), I have found that the gem was not published to  `https://rubygems.org/search?page=2&query=opentelemetry`

I saw the files as it seemed to be pure ruby and believe it is fully compatible to jruby should the gem be built and published. This is will be an easy task for the owners of this library. Simply do the following: 

```bash
# navigate to the folder 
# https://github.com/open-telemetry/opentelemetry-ruby/tree/main/propagator/xray
jruby -S gem build opentelemetry-propagator-xray.gemspec 
jruby -S gem install opentelemetry-propagator-xray-0.16.0.gem
gem list 

...
opentelemetry-propagator-xray (0.16.0)
opentelemetry-sdk (0.16.0)
pact (1.51.1)
...

gem push opentelemetry-propagator-xray-0.16.0.gem

# login to rubygems.org
```

Done.

**Comment by schanjr** [comment]
@fbogsany @robertlaurin @mwear @dazuma

**Comment by schanjr** [comment]
Btw, please change this line 
https://github.com/open-telemetry/opentelemetry-ruby/blob/main/propagator/xray/opentelemetry-propagator-xray.gemspec#L31
to this: 
  spec.add_development_dependency 'bundler', '>= 1.17'

**Comment by fbogsany** [comment]
We simply haven't cut a release since Xray propagator support was added. We anticipated pushing out a 1.0 Release Candidate 🔜 , so an interim release wasn't a priority. That process has dragged on long enough that we should push out a release with what's currently in HEAD. If you have suggestions for fixes to the Xray propagator before then, can you please open a PR with those changes?

**Comment by fbogsany** [comment]
Released with 0.17.0

**Comment by schanjr** [comment]
Was able to install the gem. Thanks for the responsiveness!
